components/02-molecules/cards/reference-card/_yds-reference-card.scss

```diff
@@ -79,7 +79,12 @@
   > *:not(:first-child) {
     margin-left: var(--size-spacing-5);
     padding-left: var(--size-spacing-5);
-    border-left: var(--border-thickness-1) solid;
+
+    &::before {
+      position: relative;
+      content: '|';
+      right: var(--size-spacing-5);
+    }
```

Comment:
This one still has the wrong `position` and `right` values. Please add `position: relative` above, and change this to `absolute` and the change `right` to `left` with the value to match the other one.